Sources/Basic/FSProxy.swift

```diff
@@ -268,19 +268,19 @@ private class LocalFS: FSProxy {
 //
 // FIXME: This class does not yet support concurrent mutation safely.
 public class PseudoFS: FSProxy {
-    private class Node {
+    fileprivate class Node {
```

Comment:
I remember thinking that too.  Probably a bogus diagnostic at one point.  I'll change it